.github/ISSUE_TEMPLATE/support---question-and-answer.md

```diff
@@ -10,3 +10,24 @@ If you have a feature request, please open up a [Feature Request](https://github
 You can find information about general PGO [support](https://access.crunchydata.com/documentation/postgres-operator/latest/support/) at:
 
 [https://access.crunchydata.com/documentation/postgres-operator/latest/support/](https://access.crunchydata.com/documentation/postgres-operator/latest/support/)
+
+For questions that are neither bugs nor feature requests, please be sure to
+
+- [ ] Provide information about your environment (see below for more information).
+- [ ] Provide any steps or other relevant details related to your question.
+- [ ] Attach logs, where applicable. Please do not attach screenshots showing logs unless you are unable to copy and paste the log data.
+- [ ] Ensure any code / output examples are [properly formatted](https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#quoting-code) for legibility.
+
+Besides Pod logs, logs may also be found in the `/pgdata/<CLUSTERNAME>/pg_log` directory on your Postgres instance.
+
+If you are looking for [general support](https://access.crunchydata.com/documentation/postgres-operator/latest/support/), please view the [support](https://access.crunchydata.com/documentation/postgres-operator/latest/support/) page for where you can ask questions.
```

Comment:
Seems to repeat lines 10-12, but then again, sometimes repetition is good, and this might be one of those times...

Reply:
Yeah, I could go either way, but I figure it's not a terrible thing to repeat. Mainly because the first section is sort of a list of potential types, where the second is more related to questions specifically. :thinking: Maybe a subtitle would make things a bit clearer? I'll try that and see how it looks